odoo/api.py

```diff
@@ -1000,6 +970,104 @@ def __iter__(self):
         return iter(self.envs)
 
 
+class Cache(object):
+    """ Implementation of the cache of records. """
+    def __init__(self):
+        # {field: {record_id: {key: value}}}
+        self._data = defaultdict(lambda: defaultdict(dict))
+
+    def contains(self, field, record):
```

Comment:
I get why you're storing them that way, but wouldn't the API be more readable/sensible if it went `(record, field[, value])`?

Reply:
Ahem. Yes, you are perfectly right!